omnibus/files/private-chef-cookbooks/private-chef/libraries/elasticsearch.rb

```diff
@@ -24,8 +26,15 @@ def self.node_memory_in_units(node, _which, unit)
   # https://www.elastic.co/guide/en/elasticsearch/reference/current/heap-size.html
   def self.heap_size_default(node)
     memory = node_memory_in_units(node, :total, :mb)
-    # Note: Preflight check will error if the system memory is less than 4GB.
-    [memory / 4, 1024].max
+    maxvalue = 26_000
+    minvalue = 1000
```

Comment:
`verify_heap_size` checks against `1024` and `26 * 1024`. Maybe these should be constants, used by both routines.

Reply:
moved it up to constants